provider/aws/dist/rack.json

```diff
@@ -1513,7 +1521,7 @@
         "AutoScalingRollingUpdate": {
           "MaxBatchSize": { "Ref": "InstanceUpdateBatchSize" },
           "MinInstancesInService": "1",
-          "PauseTime" : "PT15M",
+          "PauseTime" : "PT5M",
```

Comment:
Instances that use a Run or Boot command could take longer that 5 minutes, causing the operation to fail when it's still moving along. Maybe 10 minutes?